src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -69,40 +77,84 @@
+                  <v-icon
+                    v-if="isPartyTypeOrg(orgPerson)"
+                    color="gray9"
+                  >
+                    mdi-domain
+                  </v-icon>
                 </v-col>
                 <v-col class="overflow-hidden">
-                  <p class="people-roles-title mb-1" :class="{ 'removed': wasRemoved(orgPerson)}">
+                  <p
+                    class="people-roles-title mb-1"
+                    :class="{ 'removed': wasRemoved(orgPerson)}"
+                  >
                     {{ formatName(orgPerson) }}
                   </p>
-                  <p v-if="showEmailUnderName"
+                  <p
+                    v-if="showEmailUnderName"
                     class="info-text mb-1 people-roles-email"
-                    :class="{ 'removed': wasRemoved(orgPerson)}">
+                    :class="{ 'removed': wasRemoved(orgPerson)}"
+                  >
                     {{ orgPerson.officer.email }}
                   </p>
                   <p
                     v-if="orgPerson.officer.identifier"
                     class="info-text mb-1 people-roles-inc-number"
-                    :class="{ 'removed': wasRemoved(orgPerson)}">
-                    Incorporation Number: {{orgPerson.officer.identifier}}
+                    :class="{ 'removed': wasRemoved(orgPerson)}"
+                  >
+                    Incorporation Number: {{ orgPerson.officer.identifier }}
                   </p>
 
                   <div v-if="!isSummaryView && orgPerson.actions">
                     <!-- show DELETED item as black on light gray -->
-                    <v-chip v-if="wasRemoved(orgPerson)" x-small label color="grey lighten-2">
+                    <v-chip
+                      v-if="wasRemoved(orgPerson)"
+                      x-small
+                      label
+                      color="grey lighten-2"
+                    >
                       REMOVED
                     </v-chip>
                     <!-- show REPLACED item separately (since item also has ADDED action) -->
-                    <v-chip v-else-if="wasReplaced(orgPerson)" x-small label color="primary" text-color="white">
+                    <v-chip
+                      v-else-if="wasReplaced(orgPerson)"
+                      x-small
+                      label
+                      color="primary"
+                      text-color="white"
+                    >
                       CHANGED
                     </v-chip>
                     <!-- all other states -->
-                    <template v-else v-for="(action, i) in orgPerson.actions">
-                      <v-chip x-small label color="primary" text-color="white" :key="`action-chip-${i}`">
+                    <template
+                      v-for="(action, i) in orgPerson.actions"
+                      v-else
+                    >
+                      <v-chip
+                        :key="`action-chip-${i}`"
```

Comment:
This is also a build/lint warning. We can fix this later.

![image](https://user-images.githubusercontent.com/10002889/234071232-340c02d3-7c9e-4519-b1eb-48c23e3b5c75.png)